zkevm-circuits/src/util/word.rs

```diff
@@ -44,6 +44,10 @@ impl<T, const N: usize> WordLimbs<T, N> {
     pub fn n() -> usize {
         N
     }
+    /// get limbs by index
+    pub fn get(&self, index: usize) -> &T {
```

Comment:
When I saw this being called I didn't know what it was by the function name.  Maybe calling this `limb` would make it more clear?

Or maybe remove this method and just use `word.limbs[index]`? Considering we have `pub limbs`

Reply:
I roughly remember, this is to avoid clone of whole limbs if we only want to fetch single limb refereence. If we took `word.limbs[index]` it compiler might treat it as move, so it make us to clone something. But I think right now there is no space to use `word.limbs[index]`

Removed this api in commit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1411/commits